modules/contact-form/grunion-contact-form.php

```diff
@@ -151,11 +151,12 @@ function __construct() {
 		// custom post type we'll use to keep copies of the feedback items
 		register_post_type( 'feedback', array(
 			'labels'            => array(
-				'name'               => __( 'Feedback', 'jetpack' ),
+				'name'               => __( 'Jetpack Feedback', 'jetpack' ),
 				'singular_name'      => __( 'Feedback', 'jetpack' ),
 				'search_items'       => __( 'Search Feedback', 'jetpack' ),
 				'not_found'          => __( 'No feedback found', 'jetpack' ),
 				'not_found_in_trash' => __( 'No feedback found', 'jetpack' ),
+				'menu_name'          => __( 'Feedback' )
```

Comment:
Could you add the `jetpack` textdomain here, so that string can be translated? 

Thank you!